### ISSUE TITLE
Documentation examples for Parquet to Delta for multiple partitions

### ISSUE THREAD

**Issue by cequencer** [delta-io/delta#378]
The current Delta Lake documentation provides example for 1 partition column in Scala.
https://docs.delta.io/latest/delta-utility.html#-convert-to-delta

```
// Convert partitioned parquet table at path '/path/to/table' and partitioned by integer column named 'part'
val partitionedDeltaTable = DeltaTable.convertToDelta(spark, "parquet.`/path/to/table`", "part int")
```

My Parquet table has 3 partition columns.  It is unclear to me if current (source code) version (0.5) of Delta supports more than 1 partition column as part of this conversion.  My Spark version is 2.4.4.  (Not sure if Spark version is an requirement or not.)

If the answer is an absolute yes, some examples to show the actual syntax would be extremely helpful.  It is unclear if the string input expects a comma delimited string, and if that string has case sensitivity checks or not.  

It is good to have clarity before converting a large parquet table with at least 3 partitions for example (year, month, day) and not have any irreversible corrupted table on the other side of that function.

**Comment by cequencer** [comment]
I found the documentation I am looking for embedded inside the code comment in Spark [StructType.scala file](https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/types/StructType.scala) that provided the example I am looking for:
```
/**
   * Creates StructType for a given DDL-formatted string, which is a comma separated list of field
   * definitions, e.g., a INT, b STRING.
   *
   * @since 2.2.0
   */
  def fromDDL(ddl: String): StructType = CatalystSqlParser.parseTableSchema(ddl)
```

I have answered my own question.  I don't know how to enhance the Delta lake documentation, so I am leaving this issue open for the project maintainers to triage it or close it.  
Hopefully this will help others in the future.
